R/forecast.R

```diff
@@ -344,6 +344,31 @@ validate_general <- function(data) {
 }
 
 
+#' Validate forecast objects for internal use
+#' @description
+#' The function makes it possible to silently validate an object. In addition,
+#' it can return a copy of the data and remove rows with missing values.
+#'
+#' @inheritParams score
+#' @param copy Logical, default is `FALSE`. If `TRUE`, a copy of the input data
+#' is created.
+#' @param na.omit Logical, default is `FALSE`. If `TRUE`, rows with missing
+#' values are removed.
+#' @importFrom data.table copy
+#' @importFrom stats na.omit
+#' @keywords internal
+validate_forecast_internal <- function(data, copy = FALSE, na.omit = FALSE) {
```

Comment:
as implemented neither of copy or na.omit are ever not true. If we are doing this then why not make these the defaults?